src/Components/SingleArticle.jsx

```diff
@@ -17,7 +18,7 @@ const SingleArticle = () => {
           .then((fetchedArticle) => {
             setArticleById(fetchedArticle)
             setIsLoading(false)
-            if (shows.length === 0) {
+            if (fetchedArticle.length === 0) {
```

Comment:
is `fetchedArticle` an object or an array? 🤔 If it's object ( which it should hopefully be ) then it won't have a length property and this will always be a false condition